### PR TITLE
[cms] Add pay invoices admin button

### PR DESCRIPTION
**Pull request by alexlyp**
This adds a button on the generate payouts admin screen that uses the pay invoices endpoints.

This typically will only be used in the event of the address watcher not properly updating an invoice to paid.

### CODE REVIEW

File: src/components/GeneratePayouts/GeneratePayoutsPage.js [hunk 1]
```diff
@@ -60,6 +64,20 @@ const GeneratePayoutsPage = ({
...
+            className="inverse"
+            onClick={e =>
+              confirmWithModal(modalTypes.CONFIRM_ACTION, {
+                message:
+                  "Are you sure you want to set all of these approved invoices to paid?"
+              }).then(confirm => confirm && onPayApprovedInvoices()) &&
+              e.preventDefault()
+            }
+            text="Set invoices to paid"
+            isLoading={false}
```
Comment (fernandoabolafio) [inline src/components/GeneratePayouts/GeneratePayoutsPage.js]: The loading should not default to `false`.  It needs to be the correct value indicating whether the paid action is loading or not.

File: src/components/GeneratePayouts/GeneratePayoutsPage.js [hunk 2]
```diff
@@ -60,6 +64,20 @@ const GeneratePayoutsPage = ({
           {payouts && <PayoutsTable payouts={payouts} />}
         </div>
         <div id="csv-hidden-div" style={{ display: "none" }} />
+        <div className=" payouts-btn-pay-invoices">
+          <ButtonWithLoadingIcon
```
Comment (fernandoabolafio) [inline src/components/GeneratePayouts/GeneratePayoutsPage.js]: As far as I can tell, this button will be visible and clickable even after all the invoices were marked as paid. Shouldn't it be rendered or enabled only if the `numOfApprovedInvoices > 0` ?

File: src/components/GeneratePayouts/GeneratePayoutsPage.js (hunk 1 above)
Comment (alexlyp) [reply src/components/GeneratePayouts/GeneratePayoutsPage.js]: Well there really isn't any loading going on, as it should be handled immediately.  Should I just change which Button I'm currently using in that case?

File: src/components/GeneratePayouts/GeneratePayoutsPage.js (hunk 2 above)
Comment (alexlyp) [reply src/components/GeneratePayouts/GeneratePayoutsPage.js]: Good point, I'll add that check.

File: src/components/GeneratePayouts/GeneratePayoutsPage.js (hunk 1 above)
Comment (fernandoabolafio) [reply src/components/GeneratePayouts/GeneratePayoutsPage.js]: The click of this button will trigger a modal, which after the confirmation will be closed and fire a request against the API, right? Normally, what we do is to use the button loading to display that the request is ongoing to give feedback for the user. This also prevents the button to be clickable while the request hasn't finished yet.

**Pull request merged**